services/user.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = User.objects.create_user(username=username, password=password)
```

Comment:
Don't forget to use get_user_model insead of User. 3rd point of code style in checklist